testing/pacparser/APKBUILD

```diff
@@ -11,12 +11,14 @@ depends=""
 makedepends="bash"
 install=""
 subpackages="$pkgname-doc $pkgname-dev"
-source="$pkgname-$pkgver.tar.gz::https://github.com/pacparser/pacparser/archive/$pkgver.tar.gz"
+source="$pkgname-$pkgver.tar.gz::https://github.com/pacparser/pacparser/archive/$pkgver.tar.gz
+spidermonkey-make.patch"
 builddir="$srcdir/$pkgname-$pkgver"
 
 prepare() {
 	cd "$builddir"
 	find . -name \*.git\* -delete
+	patch -p0 -i "$srcdir/spidermonkey-make.patch"
```

Comment:
Abuild knows how to apply patches, just add `default_prepare` to your `prepare()` function and it will automatically apply all *.patch files specified in the `source`.